data/hai/hai ships.txt

```diff
@@ -11,6 +11,72 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+ship "Anomalocaris"
+	swizzle 0
+	plural "Anomalocarises"
+	sprite "ship/hai anomalocaris"
+	thumbnail "thumbnail/hai anomalocaris"
+	attributes
+		category "Medium Warship"
+		"cost" 7400000
+		"shields" 14200
+		"hull" 6900
+		"required crew" 20
+		"bunks" 26
+		"mass" 440
+		"drag" 6
+		"heat dissipation" .75
+		"fuel capacity" 400
+		"cargo space" 121
+		"outfit space" 533
+		"weapon capacity" 192
+		"engine capacity" 93
+		# Disabled faster due to the ship's sea time, which the Unfettered know about
+		"threshold percentage" 0.4
+		weapon
+			"blast radius" 84
+			"shield damage" 840
+			"hull damage" 420
+			"hit force" 1260
+	outfits
+		"Korath Detainer" 2
+		"Bullfrog Anti-Missile" 4
+		"Hai Williwaw Cooling"
+		"Boulder Reactor"
+		"Value Detector"
+		"Asteroid Scanner"
+		"Hai Valley Batteries"
+		"Hai Diamond Regenerator"
+		`"Biroo" Atomic Thruster`
+		`"Bondir" Atomic Steering`
+		"Hyperdrive"
+		"Pulse Rifle" 20
+
+	gun -24 -89.5
+	gun 24 -89.5
+	turret -24 -89.5
+	turret 24 -89.5
+	turret 0 60
+	turret 0.5 -32.5
+	engine -50 15
+	engine 50 15
+	"steering engine" -50 15
+		zoom 0.5
+		right
+	"steering engine" 50.5 15
+		zoom 0.5
+		left
+	leak "leak" 40 50
+	leak "flame" 40 30
+	explode "tiny explosion" 24
+	explode "small explosion" 50
+	explode "medium explosion" 30
+	explode "large explosion" 14
+	"final explode" "final explosion medium"
+	description "This ship is rumored to be one of the first agile warships the Hai made, when conflicts started to erupt with the Sheragi. The high cargo capacity and low engine space was probably due to the first designs of the ship being intended for mining or freighting instead, but it was useful to collect loot from the enemy ships and carry ammo as well as escorting more vulnerable ships. Vessels from that time used a more expensive but also more durable material, allowing this ship to take quite a pounding, although its time under water may have damaged it."
```

Comment:
```suggestion
	description "First created during the Hai's initial conflicts with the Sheragi, the Anomalocaris was supposedly one of the most agile warships of its time. Its high cargo capacity and low engine space suggest the first designs of the ship were intended for mining or freighting, but this also made it useful for collecting loot from enemy ships and carrying ammunition. Vessels from that time used a more expensive but also more durable material, allowing this ship to take quite a pounding, although its time under water may have damaged it."
```

Reply:
idk if I'd say "of it's time" bc it's not that agile and they quickly made more focused agile warships I imagine - such as the lightning bug, for instance

Reply:
```suggestion
	description "First created during the Hai's initial conflicts with the Sheragi, the Anomalocaris was supposedly one of the most agile Hai warships of its time. Its high cargo capacity and low engine space suggest the first designs of the ship were intended for mining or freighting, but this also made it useful for collecting loot from enemy ships and carrying ammunition. Vessels from that time used a more expensive but also more durable material, allowing this ship to take quite a pounding, although its time under water may have damaged it."
```
How about this?

Reply:
collecting loot could be shortened to looting no?